src/components/landing/Landing.tsx

```diff
@@ -3,14 +3,20 @@ import { Link } from "react-router-dom";
 import styled, { AnyStyledComponent } from "styled-components";
 
 const Land: AnyStyledComponent = styled.div`
-  height: 100%;
-  overflow-x: hidden;
-  background-color: #454c60;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #dce1ee;
+  height: 100vh;
 `;
 
-const Main: AnyStyledComponent = styled.main`
-  height: 350px;
-  color: white;
+const Main: AnyStyledComponent = styled.div`
```

Comment:
if this is truly the main you would think you would use the semantic element

Reply:
but i don't think this is actually main, so you might consider changing the title